src/creds.rs

```diff
@@ -259,3 +193,48 @@ fn test_protocol_version() {
     assert_eq!(mls10_e[0], 1);
     assert_eq!(default_e[0], 1);
 }
+
+/// This struct contains a credential and the corresponding private key.
+pub struct CredentialBundle {
+    credential: Credential,
+    signature_private_key: SignaturePrivateKey,
```

Comment:
Maybe `signature_key`? Is a little shorter.

Reply:
For the record: We discussed this in a call and decided to leave the name as-is to avoid confusion with the public part of the signature key. For example in `BasicCredential`, there is a field `signature_key`, which denotes the public part of the signature key.